scripts/add-toc.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
```

Comment:
The above line is not necessary if the file path is passed to the `node` argument :+1: